PyDMXControl/profiles/defaults/_Fixture.py

```diff
@@ -56,6 +56,9 @@ def unpark(self):
 
 class FixtureHelpers:
 
+    # Store the callbacks
+    local_callback_list = []
```

Comment:
Should this be `__local_callback_list` so that is it private?

Reply:
Good idea, will make that change.